CMakeLists.txt

```diff
@@ -74,10 +74,10 @@ option(use_prov_client "Enable provisioning client" OFF)
 option(use_tpm_simulator "tpm simulator type of hsm used with the provisioning client" OFF)
 option(use_edge_modules "Enable support for running modules against Azure IoT Edge" OFF)
 option(use_custom_heap "use externally defined heap functions instead of the malloc family" OFF)
-option(use_baltimore_cert "set use_baltimore_cert to ON if the Baltimore cert is to be used, set to OFF to not use it" OFF)
```

Comment:
removing use_baltimore_cert will brake customers scripts?

Reply:
Good point. We may not be able to remove the existing options.

Reply:
I did think about that and there are 3 reasons to remove this:
1. Breaking is a good thing in this case because a breaking change will occur in the future as the CA gets changed.
2. Jeremy just added the individual certificate selections (use_samples_certs must be present for any of these to work) last year or late 2019 for Arduino/ESP.
3. CMake options are not contractual, neither is our build system.